hackathon_site/hackathon_site/settings/__init__.py

```diff
@@ -169,6 +170,7 @@
     "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.LimitOffsetPagination",
     "DEFAULT_PERMISSION_CLASSES": ["rest_framework.permissions.IsAuthenticated"],
     "PAGE_SIZE": 100,
+    "DEFAULT_FILTER_BACKENDS": ("django_filters.rest_framework.DjangoFilterBackend",),
```

Comment:
Just for consistency with the other keys, make this a list instead of a tuple.

Normally I prefer to use tuples for stuff in views/serializers, since they are immutable. But for the settings file, in theory we might want to mutate things from the CI overrides settings. Highly unlikely we'll ever do that for this setting, so mostly just to be consistent.